tasks/1.js

```diff
@@ -0,0 +1,23 @@
+function rangeArr(a, b) {
+  const array = [];
```

Comment:
Checkout ES6 `Array.from`

Reply:
Yeah, the ES6 version expects to use a newer API, in this case both `for` and `while` loop variants are old approaches. 😉

Reply:
But task descriptions says **Do not use any ES6 methods**. I am confused :))

Reply:
Task description asks for two versions, one without ES6 and one utilising ES6 features, that's why Zanas mentioned `Array.from`, which was introduced with ES6.

Reply:
Alright. didnt understood the task then. My bad.